VK_App/VK_App/View/GlobalGroupsViewCell.swift

```diff
@@ -20,7 +20,7 @@ final class GlobalGroupsViewCell: UITableViewCell {
 
     private func setupUIInfo(_ group: Group) {
```

Comment:
```suggestion
    private func configure(_ group: Group) {
```
Пожалуйста запомни всегда configure